src/main/kotlin/org/elm/workspace/ui/ElmWorkspaceConfigurable.kt

```diff
@@ -25,19 +26,25 @@ class ElmWorkspaceConfigurable(
 
     private val elmVersionLabel = JLabel()
     private val elmFormatLabel = JLabel()
+    private val elmFormatActivateOnSaveHookCheckbox = JCheckBox()
```

Comment:
Please rename this to something shorter (`elmFormatOnSaveCheckbox` or `formatOnSaveCheckbox`)